<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>802</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>OIV Objecten</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>281</width>
         <height>760</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="terug">
            <property name="text">
             <string>Terug</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>U voegt bouwlagen toe aan de bouwlagen:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="teken_bouwlaag">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>De nu weergegeven bouwlaag is:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="bouwlaag_min">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="bouwlaag_max">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Bouwlaag toevoegen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bouwlaag_bag">
            <property name="text">
             <string>Bouwlaag overnemen van BAG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bouwlaag_tekenen">
            <property name="text">
             <string>Bouwlaag zelf tekenen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bouwlaag_overnemen">
            <property name="text">
             <string>Bouwlaag overnemen van andere bouwlaag</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Indien u wilt kopieren van een andere bouwlaag</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>selecteer hieronder dan die bouwlaag:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="bouwlaag"/>
          </item>
          <item>
           <widget class="QLabel" name="label3">
            <property name="text">
             <string>Selecteer de lagen die u mee wilt kopieren:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="copy_afw_binnen">
            <property name="text">
             <string>Afwijkende binnendekking</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="copy_picto">
            <property name="text">
             <string>Installatietechnische veiligheidsvoorzieningen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="copy_ingang_bouwl">
            <property name="text">
             <string>Ingang bouwlaag</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="copy_label">
            <property name="text">
             <string>Labels bouwlaag</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="copy_scheidingen">
            <property name="text">
             <string>Bouwkundige veiligheidsvoorzieningen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="copy_ruimten">
            <property name="text">
             <string>Ruimten</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="copy">
            <property name="text">
             <string>Selecteer en Kopieer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>150</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
